src/MSSQL/Application.php

```diff
@@ -15,7 +15,7 @@ class Application extends \Keboola\DbWriter\Application
 {
     public function __construct(array $config, Logger $logger)
     {
-        $action = !is_null($config['action']) ?: 'run';
+        $action = $config['action'] ?? 'run';
```

Comment:
Tu je chybná podmienka, 
- .... ak je `$config['action'] === run`, 
- potom `$action = true`, 
- a v podmienke dole sa pouzije config pre `action` (kde nie su vyzadovane dalsie parametre) nie pre `run` .... 

Vysledkom bola nefungujuca validacia,
- ak nieco chybalo tak sa vyhodil `Aplication` error a nie `User` error
- zistili sme to pri spustani konfiguracie s 0 config rows, vid Slack link hore.

Pridal som k tomu aj test, ktory pred upravou nepresiel a teraz prejde.

Reply:
Rovnaka logika je aj v https://github.com/keboola/php-component/blob/49af27d934f09be2e11ea1becb2c98e75032b36b/src/Config/BaseConfig.php#L139

Kde sa vrati defaultna hodnota `run` iba ked polozka neexistuje v configu:
https://github.com/keboola/php-component/blob/49af27d934f09be2e11ea1becb2c98e75032b36b/src/Config/BaseConfig.php#L78